httptest.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type HTTPTestStep struct {
```

Comment:
can you please add doc to this structures? they have ambigious names (associated with some test purposes for me)